test/happypath.test.js

```diff
@@ -514,6 +517,18 @@ describe('Happy path scenario', function() {
 
         assert.strictEqual(bobWalletUpdates, 23)
         socket.terminate()
-    })
 
-})
\ No newline at end of file
+        // Verify AMQP messages
+        let projectFullyFundedMessages = amqpUtil.getProjectFullyFundedMessages()
+        const messageSent = amqpUtil.createFullyFundedMessage(addProjWalletTxHash)
+        expect(projectFullyFundedMessages).to.have.lengthOf(1)
+        expect(projectFullyFundedMessages).to.have.members([messageSent])
+
+        let successfullyInvestedMessages = amqpUtil.getSuccessfullyInvestedMessages()
+        const aliceMessage = amqpUtil.createSuccessfullyInvestedMessage(addAliceWalletTxHash, addProjWalletTxHash, aliceInvestmentAmount)
+        const bobMessage = amqpUtil.createSuccessfullyInvestedMessage(addBobWalletTxHash, addProjWalletTxHash, bobInvestmentAmount)
+        const janeMessage = amqpUtil.createSuccessfullyInvestedMessage(addJaneWalletTxHash, addProjWalletTxHash, janeInvestmentAmount)
+        expect(successfullyInvestedMessages).to.have.lengthOf(3)
+        expect(successfullyInvestedMessages).to.have.members([aliceMessage, bobMessage, janeMessage])
```

Comment:
all good here, just pointing out that I didn't know about the .members() function. Unordered deep equality check can definitely come in handy, nice 👍